python-sdk/src/astro/files/locations/http.py

```diff
@@ -17,6 +17,10 @@ def paths(self) -> list[str]:
         """Resolve patterns in path"""
         return [self.path]
 
+    def validate_conn(self):  # skipcq: PTC-W0049
```

Comment:
`# skipcq: PTC-W0049` shouldn't be needed if you have docstring

Reply:
<img width="1685" alt="Screenshot 2022-12-28 at 5 03 56 PM" src="https://user-images.githubusercontent.com/8670962/209841621-9b6e9b45-a017-4c5d-aee1-3c1dbbe6aed4.png">
Actually, it was complaining even when I had doc-string

Reply:
try and remove the `return`, as it is redundant

Reply:
lets see -- trying in https://github.com/astronomer/astro-sdk/pull/1478/commits/775f94713f280883cd169887064ebd60141e05dc

Reply:
yup deepsource passed for it :)